lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -266,6 +273,16 @@
             $( "#webcam_capture_button", this.el ).removeClass('is-hidden');
```

Comment:
@AlasdairSwan "this.el" in selectors is the context for traversing the DOM this is same as this.$el.find("#id")